src/controllers/tests/bundler.controller.test.ts

```diff
@@ -0,0 +1,20 @@
+import request from 'supertest';
+
+import { App } from '../../app';
+
+import { BundlerController } from '../bundler.controller';
+
+describe('BundlerController', () => {
+  describe('[POST] /bundle', () => {
+    it('should bundle files', async () => {
+      const app = new App([new BundlerController()]);
+      return request(app.getServer())
+        .post('/v1/bundle')
+        .send({
+          asyncapis: ['../../fixtures/bundler/signup.yaml', '../../fixtures/bundler/login.yaml'],
+          base: '../../fixtures/bundler/output.yaml',
```

Comment:
Fix that test because you should operate on AsyncAPI documents (as content/value) not on paths to AsyncAPI documents.